src/lib/atoms/Input.svelte

```diff
@@ -29,7 +29,7 @@
   <input
     {...rest}
     class={cn(
-      'border-input aria-[invalid]:border-destructive [&>span]:data-placeholder:text-muted-foreground smText flex h-10 w-full max-w-[388px] items-center justify-between rounded-md border bg-white px-3 py-2 focus-visible:outline-hidden disabled:cursor-not-allowed disabled:opacity-50 [&>span]:line-clamp-1',
+      'aria-[invalid]:border-destructive [&>span]:data-placeholder:text-muted-foreground smText focus-visible:outline-primary flex h-10 w-full max-w-[388px] items-center justify-between rounded-md border bg-white px-3 py-2 disabled:cursor-not-allowed disabled:opacity-50 [&>span]:line-clamp-1',
```

Comment:
Looks like in Firefox and Safari this also requires: `focus-visible:outline-solid` Might also need `focus-visible:outline-2` to make sure all browsers show the 2px outline.